src/client/src/ui/analytics/views/analytics.scss

```diff
@@ -33,90 +35,113 @@ $default-padding: 10px;
   padding: 10px;
 }
 
+.analytics__header .header-bold{
+  font-weight: bold;
+}
+
+.analytics__header .block{
+  display: block;
+  padding-bottom: 2px;
+}
+
 .analytics__chart-container {
   position: relative;
   padding: $default-padding;
   background: $gray-dark;
 
-  .nvd3 g.nv-groups  {
-    .nv-bar {
-      rect {
-        fill-opacity: .3;
-        stroke: $stroke-color;
-        stroke-width: .8;
-        stroke-opacity: .4;
-      }
-      &.analytics__chart-container--positive {
-        fill: $positive-color;
-        text {
-          fill: $gray-light;
-        }
-      }
-      text {
-        font-size: 8px;
-        fill: $gray-lighter;
-        font-weight: normal;
-      }
-      &.analytics__chart-container--negative {
-        fill: $negative-color;
-      }
-      &:hover rect {
-        stroke-opacity: .5;
-        fill-opacity: .7;
-      }
-    }
-  }
-
-  .nv-controlsWrap.nvd3-svg, .nv-axisMin-y {
-    display: none;
-  }
-
   .nv-lineChart {
     .nv-axis.nv-y {
       text {
-        font-size: 8px;
+        font-size:10px;
       }
     }
 
     .nv-axis.nv-x {
       text {
-        font-size: 8px;
+        font-size: 10px;
       }
     }
   }
 }
 
-.analytics__buttons {
+.analytics__barchart-container{
+  color:$gray-light;
+  vertical-align: middle;
+  padding-top: 2px;
+  padding-bottom: 12px;
+  &:hover{
+    color:$gray-lighter
+  }
+}
+
+.analytics__barchart-bar{
+  height: $bar-height;
+}
+
+.analytics__barchart-bar .background{
+  fill: #414D5C;
+  height: $bar-height;
+}
+
+.analytics__barchart-bar .indicator{
+  stroke: $gray-darker;
+  stroke-width: 1px;
+  height: $bar-height;
+}
+
+.analytics__barchart-bar .green{
+  fill: #3FCF1E;
+}
+
+.analytics__barchart-bar .red{
+  fill: #CC3350;
+}
+
+.analytics__barchart-label{
+  font-size: 18px;
+  display: block;
+  display: inline;
+}
+
+
+.analytics__barchart-container .barchart-amount{
+  visibility: hidden;
```

Comment:
with BEM notation, every css class should start with the name of the component (ie no nesting needed). So in this can, that could be .analytics__bar-chart-amount